internal/cfgmgr/cfgmgr.go

```diff
@@ -0,0 +1,149 @@
+package cfgmgr
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf16"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+	errorspkg "github.com/pkg/errors"
```

Comment:
Curious, why the rename

Reply:
This was probably carry over from a previous iteration where I was using the golang "errors" and "github.com/pkg/errors" package, so I renamed the latter